.gitignore

```diff
@@ -1,3 +1,4 @@
+.idea
```

Comment:
This looks like an IDE-specific file, that is best left in your personal `.gitignore`. The other files ignored below actually pertain to the project and its tooling.

See: https://tekin.co.uk/2020/03/maintain-a-global-git-ignore-file

Reply:
I agree, but it also prevents other developers to accidentally add those files who haven't set up a global ignore file. I see this ignore in almost every project that I use because Jetbrains IDEs are widespread.

Reply:
@jmikola ping

Reply:
Thanks for updating the test files. I will try to make time to merge this within the next few days.

I still plan to omit this `.gitignore` change, but you don't need to modify the PR further as I can take care of that at merge time. With respect to this project, I'm confident that we'd catch an inadvertently committed `.idea` file from making its way through a PR, and if that ever happened the review itself would be a learning opportunity to suggest the contributor tune their personal `.gitignore` file.